CSLabs.Api/Models/Maintenance.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace CSLabs.Api.Models
+{
+    public class Maintenance
```

Comment:
@Cliftonz How does this relate to what Alex is doing with the alerts? Should they be the same model?

Reply:
@jasekiw @AStiner No, I do not think they should be. System Messages can be admin messages but some are from maintenance windows scheduled in the future.
Thus I would say that there should be a link but they should be different.

Reply:
@Cliftonz I'm having trouble seeing the difference here. What functionality difference is there between the SystemMessage and a maintenance window, they both have a start and end time. It almost seems like a maintenance would be a certain type of SystemMessage.